presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergSystemTablesHadoop.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.iceberg;
+
+import com.facebook.presto.testing.QueryRunner;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+
+import static com.facebook.presto.iceberg.CatalogType.HADOOP;
+
+public class TestIcebergSystemTablesHadoop
+        extends TestIcebergSystemTables
+{
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        return IcebergQueryRunner.createIcebergQueryRunner(ImmutableMap.of(), HADOOP, ImmutableMap.of());
+    }
+
+    @Test
+    public void testPropertiesTableWithSpecifiedDataWriteLocation()
```

Comment:
Hmm this is getting me thinking that maybe we ought to have a test class which creates a query runner configured with _all_ of the catalog types. Then we don't have to create new test classes which have the overhead of starting new query runners all the time.

I think this is fine for the PR, but I am concerned that we're creating a lot of overhead in testing for these different config scenarios. I think it might be worth an effort to try to consolidate to reduce the CI times.